### PR TITLE
CASMPET-6619: prereqs script cleanup

### PR DESCRIPTION
**Pull request by bo-quan**
# Description
Added a few clean-ups to prerequisites script:
- Remove spire upgrade
- Remove VCS backup
- Consistent logging for cert-manager & cray-nls upgrade
- Install platform-utils, goss-servers rpms and restart goss-servers service

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: upgrade/scripts/upgrade/prerequisites.sh [hunk 1]
```diff
@@ -1183,6 +1094,8 @@ if [[ ${state_recorded} == "0" ]]; then
   {
     export PDSH_SSH_ARGS_APPEND="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
     rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*csm-testing\*.rpm | sort -V | tail -1)"
+    rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*platform-utils\*.rpm | sort -V | tail -1)"
```
Comment (johren-hpe) [inline upgrade/scripts/upgrade/prerequisites.sh:1097]: Should we also be updating goss-servers to make sure it matches csm-testing?

Comment (johren-hpe) [reply upgrade/scripts/upgrade/prerequisites.sh:1097]: Example from surtur:
```
ncn-m001:~ # rpm -qa | grep csm-testing
csm-testing-1.16.36-1.noarch
ncn-m001:~ # rpm -qa | grep goss-servers
goss-servers-1.15.46-1.noarch
```

Comment (kimjensen-hpe) [reply upgrade/scripts/upgrade/prerequisites.sh:1097]: I was just going to suggest the same based on https://github.com/Cray-HPE/node-images/blob/main/vendor/github.com/Cray-HPE/metal-provision/roles/node_images_ncn_common/files/scripts/metal/metal-lib.sh#L499

Comment (bo-quan) [reply upgrade/scripts/upgrade/prerequisites.sh:1097]: Thanks. Fixed.

**Pull request merged**